consensus/src/consensusdb/mod.rs

```diff
@@ -36,7 +36,10 @@ impl ConsensusDB {
 
         let path = db_root_path.as_ref().join("consensusdb");
         let instant = Instant::now();
-        let db = DB::open(path.clone(), "consensus", column_families)
+        let mut opts = Options::default();
+        opts.create_if_missing(true);
+        opts.create_missing_column_families(true);
```

Comment:
these default as true, why are we modifying it?

Reply:
default is false. Here ConsensusDB take raw schemaDB so it doesn't do the conversion.